### PR TITLE
Jib Builder: add more tests

### PR DESCRIPTION
**Pull request by dgageot**
Signed-off-by: David Gageot <david@gageot.net>

### CODE REVIEW

File: pkg/skaffold/build/local/local.go [hunk 1]
```diff
@@ -35,6 +35,11 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// For testing
+var (
```
Comment (loosebazooka) [inline pkg/skaffold/build/local/local.go]: I don't really understand this specific part? Does this just become a variable in the scope of the local pkg? Was there some problem with using the direct reference (`docker.RemoteDigest`) in the jib files?

Comment (dgageot) [reply pkg/skaffold/build/local/local.go]: This is the simplest way to mock the call to `docker.RemoteDigest`. We use that pattern in. a lot of places.

**Comment by dgageot** [comment]
@loosebazooka Looks like the changes to the owners file are still not enough

**Comment by TadCordle** [comment]
Might be because you made a change to `local.go`?

**Comment by loosebazooka** [comment]
@dgageot as a followup perhaps change all of the `docker.RemoteDigest` -> `getRemoteDigest` ?
```
build/local/bazel.go:   return docker.RemoteDigest(tag, insecureRegistries)
build/local/custom.go:          return docker.RemoteDigest(tag, b.insecureRegistries)
build/cache/save.go:            return docker.RemoteDigest(img, c.insecureRegistries)
build/cache/save.go:            return docker.RemoteDigest(img, c.insecureRegistries)
build/cache/cache.go:   remoteDigest    = docker.RemoteDigest
build/cluster/kaniko.go:        return docker.RemoteDigest(tag, b.insecureRegistries)
build/cluster/cluster.go:       return docker.RemoteDigest(tag, b.insecureRegistries)
```

**Comment by dgageot** [comment]
Well, let’s do it only when a test requires it.

**Pull request merged**